topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.md

```diff
@@ -0,0 +1,355 @@
+
+> ### {% icon hands_on %} Hands-on: Filter sequence with fastp
+>
+> 1. {% tool [fastp](toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/0.20.1+galaxy0) %} with the following parameters:
+>    - *"Single-end or paired reads"*: `Single-end`
+     - {% icon param-collection %} *"Dataset collection"*: `soil collection trimmed`
+>        - In *"Adapter Trimming Options"*:
+>            - *"Disable adapter trimming"*: `Yes`
+>    - In *"Filter Options"*:
+>        - In *"Quality filtering options"*:
+>            - *"Qualified quality phred"*: `9`
+>        - In *"Length filtering options"*:
+>            - *"Length required"*: `1000`
+>            - *"Maximum length"*: `2000`
+>    - In *"Read Modification Options"*:
+>        - *"PolyG tail trimming"*: `Disable polyG tail trimming`
+>
+> 2. Rename the output as `soil collection processed`
+>
+>
+{: .hands_on}
+
+# Re-evaluate datasets quality
+
+After processing the sequences, we are going to analyze them again using __FastQC__ and __MultiQC__ to see if we have managed to correct the anomalies that we had detected.
+
+> ### {% icon hands_on %} Hands-on: Quality check
+>
+> 1. {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72+galaxy1) %} with the following parameters:
+>    - {% icon param-collection %} *"Dataset collection"*: `soil collection processed` 
+>
+> 4. Rename the outputs as `FastQC processed: Raw` and `FastQC processed: Web` 
+> 3. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy1) %} with the following parameters:
+>    - In *"Results"*:
+>      - *"Which tool was used generate logs?"*: `FastQC`
+>      - {% icon param-collection %} *"Dataset collection"*:
+>
+>		- `FastQC processed: Raw`
+> 4. Click on the {% icon galaxy-eye %} (eye) icon and inspect the generated HTML file
+> 
+>
+{: .hands_on}
+
+We can verify that after processing the samples, the GC content presents a unimodal distribution, which indicates that the anomalies in the sequences have been successfully eliminated.
+
+![FasTQC plots GC](../../images/metagenomics-nanopore/fasqc_per_sequence_content_plot_clean.png "Per sequence GC content in processed samples")
+
+# Assign taxonomic classifications
+
+One of the key steps in metagenomic data analysis is to identify the taxon to which the individual reads belong. Taxonomic classification tools are based on microbial genome databases to identify the origin of each sequence.
+
+## Taxonomic classification with Kraken2
+
+To perform the taxonomic classification we will use __Kraken2__ ({% cite Wood_2019 %}), a tool that assigns labels by examining the k-mers within a sequence and querying them in a database that uses a compact hash table for algorithm efficiency ({% cite deKoning2020 %}).
+
+![Taxonomic classification](../../images/metagenomics-nanopore/kmers-kraken.jpg "Kramer2 sequence classification algorithm ({% cite Wood2014 %})")
```

Comment:
I would say something about the algorithm if you want to include this figure.

Reply:
@willemdek11 I included some information about that.